Engine/fgens.c

```diff
@@ -1769,7 +1744,7 @@ static int gen30(FGDATA *ff, FUNC *ftp)
       x[i] = FL(0.0);
     x[1] = x[l1];
     x[l1] = x[l1 + 1] = FL(0.0);
-    csound->InverseRealFFT(csound, x, l1);
+    csound->RealFFT(csound, csound->RealFFTSetup(csound,l1,FFT_FWD), x);
```

Comment:
Checking here: the older code called csound->InverseRealFFT(), but this is now calling csound->RealFFT(). Is this correct?